concourse/model/traits/image_scan.py

```diff
@@ -201,7 +222,14 @@ class Notify(enum.Enum):
         default=(),
         type=typing.Set[str],
         doc='if present, generated build steps depend on those generated from specified traits',
-    )
+    ),
+    # TODO required
```

Comment:
well.. .change to required, then?

Reply:
Wanted to discuss in first place since some pipelines might break.